blueprints/addon/index.js

```diff
@@ -16,6 +16,10 @@ const stringifyAndNormalize = require('../../lib/utilities/stringify-and-normali
 module.exports = {
   description: 'The default blueprint for ember-cli addons.',
 
+  filesToRemove: [
```

Comment:
is having this in the app blueprint enough? I never tested an addon for existing filesToRemove.